src/sdk/pynni/nni/ppo_tuner/__init__.py

```diff
@@ -0,0 +1 @@
+from .ppo_tuner import PPOTuner
```

Comment:
Missing new line at end of file. I suppose this can't pass pylint.

Reply:
fixed, thanks